### PR TITLE
Mon 3834 display datetime in csv export

### PR DESCRIPTION
**Pull request by sc979**
<h1> Pull Request Template </h1>

<h2> Description </h2>

Correct the date in the exported csv files :
1 - Convert the timestamps to formatted dates/times for each occurences.
2 - Homogenize the formatted start and end date.
3 - improve string sent from request security
4 - setting a default start date if the user didn't chose a period, nor a start date (to avoid to display/export logs since 1/1/1970 -> timestamp = 0 )

Fixes # (support)

<h2> Type of change </h2>

- [x] Patch fixing an issue (non-breaking change)

<h2> Target serie </h2>

- [x] 18.10.x
- [x] 19.04.x (master)

<h2> How this pull request can be tested ? </h2>

Please describe the **procedure** to verify that the goal of the PR is matched. Provide clear instructions so that it can be **correctly tested**.

1 - Go to Monitoring -> event logs, choose a resource -> export the csv -> then check the csv output
2 - Go to Monitoring -> event logs, choose a resource and a start date -> export the csv -> then check the csv output
3 - Go to Monitoring -> event logs, choose a resource and a period -> export the csv -> then check the csv output

do the same for Monitoring -> system logs

<h2> Checklist </h2>

<h5> Community contributors & Centreon team </h5>

- [x] I followed the **coding style guidelines** provided by Centreon
- [x] I have commented my code, especially new **classes**, **functions** or any **legacy code** modified. (***docblock***)
- [x] I have commented my code, especially **hard-to-understand areas** of the PR.
- [x]  I have  rebased my development branch on the base branch (master, maintenance).
- [x] I have made sure that **acceptance tests** related to the story are successful (**local and CI**)

### CODE REVIEW

**Comment by sc979** [comment]
indeed, they are (in the foreach)

File: www/include/eventLogs/export/data.php [hunk 1]
```diff
@@ -126,8 +130,8 @@
         _("Contact") . ";" .
         _("Cmd") . "\n";
     foreach ($xml->line as $line) {
-        echo $line->date . ";" .
-            $line->time . ";" .
+        echo date(_('Y/m/d'), intval($line->date)) . ";" .
```
Comment (loiclau) [inline www/include/eventLogs/export/data.php]:
```suggestion
        echo date(_('Y/m/d'), (int)$line->date) . ";" .
```

File: www/include/eventLogs/export/data.php [hunk 2]
```diff
@@ -126,8 +130,8 @@
...
+            date(_('H:i:s'), intval($line->time)) . ";" .
```
Comment (loiclau) [inline www/include/eventLogs/export/data.php]:
```suggestion
            date(_('H:i:s'), (int)$line->time) . ";" .
```

File: www/include/eventLogs/export/data.php [hunk 3]
```diff
@@ -139,16 +143,31 @@
             $line->contact_cmd . "\n";
     }
 } else {
-    echo _("Begin date") . "; " . _("End date") . ";\n";
-    echo date(_('m/d/Y (H:i:s)'), intval($xml->infos->start)) . ";" .
-        date(_('m/d/Y (H:i:s)'), intval($xml->infos->end)) . "\n";
+    echo _("Begin date") . "; "
+        . _("End date") . ";\n";
+    echo date(_('Y/m/d (H:i:s)'), intval($xml->infos->start)) . ";"
```
Comment (loiclau) [inline www/include/eventLogs/export/data.php]:
```suggestion
    echo date(_('Y/m/d (H:i:s)'), (int)$xml->infos->start) . ";"
```

File: www/include/eventLogs/export/data.php [hunk 4]
```diff
@@ -139,16 +143,31 @@
...
+        . date(_('Y/m/d (H:i:s)'), intval($xml->infos->end)) . "\n";
```
Comment (loiclau) [inline www/include/eventLogs/export/data.php]:
```suggestion
        . date(_('Y/m/d (H:i:s)'), (int)$xml->infos->end) . "\n";
```

File: www/include/eventLogs/export/data.php [hunk 5]
```diff
@@ -139,16 +143,31 @@
...
+
+    echo _("Day") . ";"
+        . _("Time") . ";"
+        . _("Poller") . ";"
+        . _("Output") . "; " . "\n";
     foreach ($xml->line as $line) {
-        echo "\"" . $line->date . "\";\"" . $line->time . "\";\"" . $line->poller .
-            "\";\"" . $line->output . "\";" . "\n";
+        echo "\"" .
+            date(_('Y/m/d'), intval($line->date)) . "\";\"" .
```
Comment (loiclau) [inline www/include/eventLogs/export/data.php]:
```suggestion
            date(_('Y/m/d'), (int)$line->date) . "\";\"" .
```

File: www/include/eventLogs/export/data.php [hunk 6]
```diff
@@ -139,16 +143,31 @@
...
+            date(_('H:i:s'), intval($line->time)) . "\";\"" .
```
Comment (loiclau) [inline www/include/eventLogs/export/data.php]:
```suggestion
            date(_('H:i:s'), (int)$line->time) . "\";\"" .
```

**Pull request merged**